.travis.yml

```diff
@@ -12,3 +12,18 @@ install:
 # command to run tests
 script:
   - make test
+
+before_deploy: sed -i "s/__version__ = \".*\"/__version__ = \"$TRAVIS_TAG\"/g" azkaban_cli/__version__.py
+
+deploy:
+  provider: pypi
+  user: "Your username"
+  password:
+    secure: "Your encrypted password"
```

Comment:
If I really can use Travis environment variables here, then come back with password variable. I will encrypt it using Travis and add as an environment variable there